src/main/java/com/intuit/graphql/filter/visitors/SQLExpressionVisitor.java

```diff
@@ -162,7 +162,11 @@ public String visitExpressionValue(ExpressionValue<? extends Comparable> value,
         Operator operator = operatorStack.pop();
         List<? extends Comparable> expressionValues  = value.getValues();
 
-        if (operator == Operator.CONTAINS || operator == Operator.STARTS || operator == Operator.ENDS ) {
+        if (operator == Operator.STARTS) {
+            expressionBuilder.append("'").append(expressionValues.get(0)).append("%").append("'");
+        } else if (operator == Operator.ENDS) {
```

Comment:
Lets add tests for these operators in the SQLExpressionTest file.

Reply:
👍  Added a test for each of those.